Colours_And_Fonts_style_v1.css

```diff
@@ -62,12 +127,13 @@ p {
 /* CSS for styling my Navigation/Menu */
 .nav {
   box-sizing: border-box;
-  background-color: #2c7da0; /*Blue*/
+  background-color: #3750A5; /*Blue*/

```

Comment:
I changed the colour from a greeny blue colour to a navy blue colour as the greeny blue colour was too dark and it didn't look that great on the webite as it didn't incoperate the Tawa College Webaite so I changed the colour to a anavy blue to make it match the header.